README.md

```diff
@@ -24,56 +24,66 @@ In this challenge, create a web page that presents a styled list of Star Wars ch
 
 Demonstrate your understanding of this Sprint's concepts by answering the following free-form questions. Edit this document to include your answers after each question. Make sure to leave a blank line above and below your answer so it is clear and easy to read by your Team Lead.
 
-- [ ] What is React JS and what problems does it try and solve? Support your answer with concepts introduced in class and from your personal research on the web.
+- [x] What is React JS and what problems does it try and solve? Support your answer with concepts introduced in class and from your personal research on the web.
 
-- [ ] What does it mean to _think_ in react?
+Some problems that react solves are faster rendering, helpful developer tools, guarantees a stable code, and boosts productivity.
 
-- [ ] Describe state.
+- [x] What does it mean to _think_ in react?
 
-- [ ] Describe props.
+To think in react, means to think about the applications as they are built out.
 
-- [ ] What are side effects, and how do you sync effects in a React component to state or prop changes?
+- [x] Describe state.
+
+State within react is the heart of react. It allows for change of certain elements within an application.
+
+- [x] Describe props.
+
+Properties are attributes that can be given to child components from a parent component. Information that is passed down a virtual tree.
+
+- [x] What are side effects, and how do you sync effects in a React component to state or prop changes?
+
+A side effect is anything that affects something outside the scope of the function that is executed.
```

Comment:
Included in this project as an `Answers.md` I would recommend checking future projects for similar files and answer questions in the place that makes most sense given a projects structure in the future.

Alternatively you could have deleted `Answers.md` and answered them here if you wanted to keep this similar to previous projects, that would be up to you to decide which you like better but I would recommend not having `Answers.md` be blank.